main.py

```diff
@@ -137,7 +137,7 @@ def validate_scenario_list(scenario_params_list, experiment_path):
         current_scenario = scenario.Scenario(scenario_params, experiment_path, is_dry_run=True)
         current_scenario.instantiate_scenario_partners()
 
-        if isinstance(current_scenario.samples_split_option, list):
+        if isinstance(current_scenario.samples_split_description, list):
```

Comment:
`if self.samples_split_type == "advanced"`
(like you did in `run_scenario()`)

Reply:
Oh, I missed that one